### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by Yazino12**
### In this pull:

- Refactor the code to use **Javascript classes**
- Add `Class`, `Constructor `and its `Methods`
- Modify `Add `and `Remove` functions to methods in the class
- Modify `CSS `for the final look

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] when submitting the form the user receives the below error, please make sure to fix it before requesting another review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yazino12** [comment]
@mikethreels would you please add the error description, I guess you miss that?

**Comment by Yazino12** [comment]
Hello @mikethreels, I've seen the error you mentioned, and It would be helpful if you can recommend ways to fix it as this error is coming from github pages.

File: index.js [hunk 1]
```diff
@@ -1,65 +1,75 @@
...
-
-// PRESERVE DATA IN THE BROWSER (LOCAL STORAGE)
-
-const fillForm = localStorage.getItem('bookData');
-
-if (fillForm) {
-  const data = JSON.parse(localStorage.getItem('bookData'));
-
-  bookCollection.push(...data);
+function Book(title, author) {
```
Comment (Meltrust) [inline index.js:3]: - [x] OPTIONAL: I recommend changing this "function" keyword to a class with a constructor.

**Review by Meltrust — APPROVED** [review]
_Note:  Reviewed **_both_** locally and on Github deployment and the project is working now with no console errors._  

Hi @Yazino12,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Yazino12) [reply index.js:3]: @Meltrust Thanks for the time. Will fix that soon.

**Pull request merged**